mod/01_rating/set/type/metric_value.rb

```diff
@@ -334,7 +332,24 @@ def special_editor? args
     end
   end
 
-  view :metric_company_add_value_editor do |_args|
+  def find_potential_sources company, metric
+    Card.search type_id: Card::SourceID,
+                right_plus: ['company', { refer_to: company }],
+                and: {
+                  right_plus: [
+                    'report_type',
+                    refer_to: {
+                      referred_to_by: metric + '+report_type' }] }
```

Comment:
does that second "refer_to" get interpreted as a hash even though there are no opening/closing curlies?